Dockerfile

```diff
@@ -6,6 +6,7 @@ WORKDIR /go/src/fleet-telemetry
 COPY . .
 ENV CGO_ENABLED=1
 
+RUN apt update -y && apt upgrade -y && apt install -y libzmq3-dev
```

Comment:
should you install zmq in base-debian11 image instead?
https://github.com/teslamotors/fleet-telemetry/blob/main/Dockerfile#L14